charts/netbox/files/ldap_config.py

```diff
@@ -61,18 +61,32 @@ def _import_group_type(group_type_name: str) -> Any | None:
 AUTH_LDAP_GROUP_SEARCH = LDAPSearch(
     AUTH_LDAP_GROUP_SEARCH_BASEDN,
     ldap.SCOPE_SUBTREE,
-    "(objectClass=" + AUTH_LDAP_GROUP_SEARCH_CLASS + ")",
+    f"(objectClass={AUTH_LDAP_GROUP_SEARCH_CLASS})",
 )
 AUTH_LDAP_GROUP_TYPE = _import_group_type(AUTH_LDAP_GROUP_TYPE)
 
 # Define a group required to login.
 AUTH_LDAP_REQUIRE_GROUP = reduce(
-    lambda x, y: x | LDAPGroupQuery(y), AUTH_LDAP_REQUIRE_GROUP_LIST, False
+    lambda query, group: query | LDAPGroupQuery(group),
+    AUTH_LDAP_REQUIRE_GROUP_LIST,
+    LDAPGroupQuery(),
```

Comment:
I tested the implementation on my site and observed the issue that any user that connects using LDAP can login and receives active/staff and superuser independent of their group assignments.

Replacing LDAPGroupQuery() with LDAPGroupQuery("cn=some-group-that-does-not-exist") resolves the issue and the group assignments are being considered. So I guess LDAPGroupQuery() with no parameters does not behave as expected in the reduce functions in ldap_config.py

Reply:
Thanks a lot for your answer. 
Can I ask you to try with the same code but with a negated initializer then? Note the tilde `~`:
```python
reduce(..., ~LDAPGroupQuery())
```

Reply:
Hi @LeoColomb ,
yeah I can confirm - the negated initializer and the empty-string argument - both resolve the issue for me.